### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by NoPro17** [FTBTeam/FTB-Modpack-Issues#1372]
### Modpack

FTB Inferno

### Modpack version

1.0.1

### Log Files

[crash-2022-10-02_06.58.21-server.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/9692959/crash-2022-10-02_06.58.21-server.txt)

### Describe the bug

When you crouch near Hexerei plants the client crashes

### Steps to reproduce

1. be by any Hexerei plant
2. crouch

### Expected behaviour

auto-bonemeal nearby

### Additional information

its an issue with the bonemeal on crouch to grow trees

**Comment by Saereth** [comment]
Need more issue on this, which hexerei plant specifically? How did you plant it etc, I'm unable to reproduce this so I need a few more specifics

**Comment by orbidlo** [comment]
From the crash log it seems it's because of PickableDoubleFlower and it's most probably ever returning bug in Hexerei, see 
https://bytemeta.vip/repo/JoeFoxe/Hexerei-1.18/issues/41 as it should have been fixed in Hexerei 0.2.1, which is weird since we are on 0.2.2

**Comment by Saereth** [comment]
yeah I assumed as much but before I report it back I need to get all the info possible so knowing which flower it was is good to know

**Comment by NoPro17** [comment]
I am currently trying to replicate it, running into issues with the Hexerei mod in general now... My survival world only has mandrake and sage right now which are from seeds work just fine, belladonna and yellow dock are the ones that i am trying to get again to double check which of those two it was, mixing cauldron is not cooperating.

**Comment by Saereth** [comment]
this may be an issue on our end then setting the block without matching a blockstate to it, I'll look into this more
